packages/mui/src/components/crud/show/index.tsx

```diff
@@ -150,7 +152,7 @@ export const Show: React.FC<ShowProps> = ({
     );
 
     return (
-        <Card {...(wrapperProps ?? {})}>
+        <Card {...(wrapperProps ?? {})} sx={{ position: "relative" }}>
```

Comment:
Could you provide a demo video using one of the examples?

Reply:
https://github.com/refinedev/refine/assets/57894795/17bde139-fc51-42ec-9b47-bacdc1caeda4

Here

Reply:
Thanks for the reivew!

Reply:
I am also thinking about adding backdropProps, so that it is possible to pass props to the <Backdrop> component, similar to other props, such as contentProps etc. I think it is a good idea. What do you think?

Reply:
Hey @issa012, thanks for the video but it looks like you've recorded the one for the `<List />` 😅

About the same line, I think if user sends `wrapperProps.sx` its overwritten by the latter. Can we update this line to also cover `wrapperProps.sx`?

```suggestion
        <Card {...(wrapperProps ?? {})} sx={{ position: "relative", ...wrapperProps?.sx }}>
```

Reply:
Hi. You're right, I recorded the wrong component. Here is the correct component.

https://github.com/refinedev/refine/assets/57894795/34812369-cc9a-4e4e-965f-505019198f8e
I have made some changes. I moved the sx={{position: "relative"}} to CardContent. It can also now be overridden. The color of the backdrop is now similar to the color when <DataGrid> is loading.

Reply:
Thank you for the update @issa012!